Descent3/game.cpp

```diff
@@ -1354,3 +1363,11 @@ void DoScreenshot() {
 
   StartTime();
 }
+
+bool ShouldCaptureMouse() {
+  // If the UI cursor is visible, then the mouse shouldn't be captured.
+  if (ui_IsCursorVisible())
```

Comment:
Side note, what controls ui_IsCursorVisible? the logic seems scatted around the codebase